janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -111,14 +111,14 @@
 
     private static final int DEFAULT_PORT = 9042;
 
+    private static Cluster cluster;
+    private static Map<String,Session> sessions = new ConcurrentHashMap<>();
```

Comment:
@sjudeng I get it, its static. You are considering the case of multiple cql store manager instances in the same JVM. When did this come up in testing?